.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,7 +10,11 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
+
```

Comment:
What do you think about switching the order around like this?

**To Reproduce**
Steps to reproduce the behavior:

Go to '...'
Click on '....'
Scroll down to '....'
See error

If you're using Hydrogen UI with NextJS, you can use our template to create a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Get started by clicking the button below:
[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/fork/github/shopify/hydrogen-ui/tree/main/apps/nextjs?file=pages%2Findex.tsx)